src/App.js

```diff
@@ -1,25 +1,22 @@
 import React from 'react';
-import logo from './logo.svg';
+import {Row, Col, Layout} from 'antd';
```

Comment:
Revisa los espaciados acá 
```suggestion
import { Row, Col, Layout } from 'antd';
```